pkg/kubelet/metrics/metrics.go

```diff
@@ -25,43 +25,74 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const kubeletSubsystem = "kubelet"
+const (
+	KubeletSubsystem              = "kubelet"
+	PodWorkerLatencyKey           = "pod_worker_latency_microseconds"
+	SyncPodsKey                   = "sync_pods_latency_microseconds"
```

Comment:
SyncPodsLatency, PodStartLatency, PodStatusLatency - generally please be more specific with keys. I wouldn't be surprised if at some point someone would want to gather number of calls, throughput or sth.